snap/snapcraft.yaml

```diff
@@ -131,8 +131,8 @@ parts:
       - mysql-server-8.0=8.0.36-0ubuntu0.22.04.1
       - mysql-router=8.0.36-0ubuntu0.22.04.1
       - mysql-shell=8.0.36+dfsg-0ubuntu0.22.04.1~ppa4
-      - prometheus-mysqld-exporter=0.14.0-0ubuntu0.22.04.1~ppa1
-      - prometheus-mysqlrouter-exporter=4.0.5-0ubuntu0.22.04.1~ppa1
+      - prometheus-mysqld-exporter=0.15.0-0ubuntu0.22.04.1~ppa1
```

Comment:
as requested, reverted to [0.14.0-0ubuntu0.22.04.1~ppa2](https://launchpad.net/~data-platform/+archive/ubuntu/mysqld-exporter/+sourcepub/15912049/+listing-archive-extra) (please notice ppa2!).

Reply:
reverted in fa4e9a8c4b064b02274bd4bfa528966a8e690469